### PR TITLE
Update Readme.md for general release

### PR DESCRIPTION
**Pull request by theJasonHelmick**

### CODE REVIEW

**Review by JamesWTruher — APPROVED** [review]
i like it!

File: README.md [hunk 1]
```diff
@@ -1,52 +1,67 @@
...
-We are approaching our general release (GA) of **PowerShell Crescendo 1.0**. We look forward to
-community feedback and suggestions. Please file issues for erroneous behavior or suggested features.
-As we stabilize for GA, we will not be taking feature PRs.
+For a detailed walkthrough using Crescendo, see this excellent blog series from Sean Wheeler -
+Thanks Sean!
+
+- [PowerShell Community](https://devblogs.microsoft.com/powershell-community/).
+- [My Crescendo journey](https://devblogs.microsoft.com/powershell-community/my-crescendo-journey/)
+- [Converting string output to objects](https://devblogs.microsoft.com/powershell-community/converting-string-output-to-objects/)
+- [A closer look at the parsing code of a Crescendo output handler](https://devblogs.microsoft.com/powershell-community/a-closer-look-at-the-parsing-code-of-a-crescendo-output-handler/)- [A closer look at the Crescendo configuration](https://devblogs.microsoft.com/powershell-community/a-closer-look-at-the-crescendo-configuration/)
```
Comment (sdwheeler) [inline README.md:54]:
```suggestion
- Crescendo on the [PowerShell Community Blog](https://devblogs.microsoft.com/powershell-community/tag/crescendo/).
```

File: README.md [hunk 2]
```diff
@@ -1,52 +1,67 @@
...
+We value your ideas and feedback and hope you will give Crescendo a try and let us know of any
+issues you find.
+
+## Release history
+
+- 12/20 - [Crescendo.Preview.1](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-1/)
+- 5/21 - [Crescendo.Preview.2](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-2/)
+- 7/21 - [Crescendo.Preview.3](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-3/)
+- 10/21 - [Crescendo.Preview.4](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-4/)
+- 12/21 - [Crescendo.RC](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-release-candidate-rc/)
```
Comment (sdwheeler) [inline README.md:67]: I think there is some value in show the release history as individual link with dates. But if you wanted to reduce this to one link you can use `https://devblogs.microsoft.com/powershell/tag/powershell-crescendo/`.

The benefit of that link, and the similar change I made for the Community blog, is that the link will include any future posts that are tagged for Crescendo. No need to update the Readme again.

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my comments

File: README.md (hunk 2 above)
Comment (theJasonHelmick) [reply README.md:64]: oh darn, now im torn

File: README.md [hunk 3]
```diff
@@ -1,52 +1,64 @@
...
```
Comment (sdwheeler) [inline README.md:64]:
```suggestion
Release announcements on the [PowerShell Blog](https://devblogs.microsoft.com/powershell/tag/powershell-crescendo/).

- 12/20 - Crescendo.Preview.1
- 05/21 - Crescendo.Preview.2
- 07/21 - Crescendo.Preview.3
- 10/21 - Crescendo.Preview.4
- 12/21 - Crescendo.RC
```

Comment (theJasonHelmick) [reply README.md:64]: @sdwheeler - ;) You did it!  That really works for me!

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**